From: Olivier Fourdan <ofourdan@redhat.com>
Date: Mon, 7 Aug 2023 15:30:05 +0200
Subject: settings: Use appropriate type for
 xwayland-allow-byte-swapped-clients

The key "xwayland-allow-byte-swapped-clients" is of type boolean, it's
not a flag.

(cherry picked from commit b9b26cba03f2eddc137a27a5dc4aa63a4253bd2d)

Forwarded: https://gitlab.gnome.org/GNOME/mutter/-/merge_requests/3156
Applied-upstream: 44.4, commit:5496bf54757a77046c531c3a9c2ecedc6ac1d1f4
---
 src/backends/meta-settings.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/backends/meta-settings.c b/src/backends/meta-settings.c
index b8b34ab..229b7f6 100644
--- a/src/backends/meta-settings.c
+++ b/src/backends/meta-settings.c
@@ -436,8 +436,8 @@ update_xwayland_allow_byte_swapped_clients (MetaSettings *settings)
 {
 
   settings->xwayland_allow_byte_swapped_clients =
-    g_settings_get_flags (settings->wayland_settings,
-                          "xwayland-allow-byte-swapped-clients");
+    g_settings_get_boolean (settings->wayland_settings,
+                            "xwayland-allow-byte-swapped-clients");
 }
 
 static void
